src/common/wallet/types.js

```diff
@@ -6,6 +6,9 @@ const WALLET_SETUP_UPDATE = 'app/wallet/setup/UPDATE';
 const WALLET_DID_CREATE = 'app/wallet/did/CREATE';
 const WALLET_DID_UPDATE = 'app/wallet/did/UPDATE';
 const WALLET_ASSOCIATE_DID_ERROR_SET = 'app/wallet/did/ASSOCIATE/ERROR';
+const WALLET_DID_ORIGIN_URL_SET = 'app/wallet/did/ORIGIN_Url/SET';
+const WALLET_START_DID_FLOW = 'app/wallet/CREATE_DID_FLOW/START';
+const WALLET_START_ACCOSIATE_DID_FLOW = 'app/wallet/ACCOSIATE_DID_FLOW/START';
```

Comment:
Did you meant WALLET_START_ASSOCIATE_DID_FLOW ?

Reply:
fixed